CMakeLists.txt

```diff
@@ -302,6 +302,11 @@ include (cmake/find_prometheus.cmake)
 include (cmake/find_tiflash_proxy.cmake)
 include (cmake/find_xxhash.cmake)
 
+if (USE_QPL AND OS_LINUX AND ARCH_AMD64 AND TIFLASH_ENABLE_AVX512_SUPPORT)
```

Comment:
Actually we don't actually need `USE_QPL`. `ENABLE_QPL` should be sufficient.

Suggest changes followed.